js/store.js

```diff
@@ -53,6 +53,37 @@ export class StoreClass {
     return undefined
   }
 
+  getMetricDrawState (metricName) {
+    for (const metricBase in this.state.allMetrics) {
+      const metricArray = []
+      if (this.state.allMetrics[metricBase].name === metricName) {
+        metricArray.push(this.state.allMetrics[metricBase].drawMin)
+        metricArray.push(this.state.allMetrics[metricBase].drawAvg)
+        metricArray.push(this.state.allMetrics[metricBase].drawMax)
+        return metricArray
+      }
+    }
+  }
+
+  checkMetricDrawState () {
+    var assume = true
+    const metricsArray = this.getAllMetrics()
+    document.getElementById('checkbox_min_max').indeterminate = false
+    metricsArray.forEach(metric => {
+      const metricDrawArray = this.getMetricDrawState(metric)
+      metricDrawArray.forEach(element => {
+        if (element !== assume) {
+          if (assume === true) {
+            assume = false
+          } else {
+            document.getElementById('checkbox_min_max').indeterminate = true
+          }
+        }
+      })
+    })
```

Comment:
This doesn't actually check what we need (and remember we need to treat avg differently than min/max).